src/uproot/behaviors/TBranch.py

```diff
@@ -2817,7 +2805,7 @@ def _regularize_expression(
                 arrays,
                 expression_context,
                 branchid_interpretation,
-                symbol_path + (symbol,),
+                (*symbol_path, symbol),
```

Comment:
That's cute: I didn't know about that. This is available in Python 3.7? I guess it is, since those tests passed.

Reply:
I think it was new in 3.0 (splat operators work in more places) - it was at least there by 3.4, for sure. It's also why `{**a, **b}` is a Python 3 way to merge dictionaries (though losing custom classes, unlike 3.9 or 3.10 with `|` on dicts).